modules/platform/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/main/java/org/nuxeo/elasticsearch/audit/ESExtendedInfo.java

```diff
@@ -37,6 +39,7 @@ public ESExtendedInfo(Serializable value) {
         this.value = value;
     }
 
+    @JsonIgnore
```

Comment:
Why? Comment maybe?

Reply:
when represented as JSON, the getId is called and throws an UnsupportedOperationException we don't want that